test/LoadTest/cache_refresh/README.md

```diff
@@ -69,25 +69,34 @@ These tests are used to verify the cache refresh in a balanced system.
   Parameters:
   ```
      <url>: host and port used by requests_listener.py script (MANDATORY)
+     -reset: is used to reset the listener previously (Optional)
```

Comment:
`-reset` parameter is optional, but if it is used the listener is reseted remotely from reqs_x_secs.sh application. Initialice the total request and tps.